specs/driver_spec.rb

```diff
@@ -77,4 +77,94 @@
       driver.average_rating.must_equal 0
     end
   end
+
+  describe "calculate_driver_revenue method" do
+    it "returns the total revenue the driver earned on 80% of the total fare after the service fee" do
+
```

Comment:
Other questions that I would like to see these tests ask, for all the wave 1 methods (on both `Driver` and `Passenger`):

- What happens if the driver or passenger has no trips?
- What happens if the driver or passenger has an incomplete trip?